src/__tests__/verify.test.js

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import Verify from '../App/Pages/verifyUser.page';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import thunk from 'redux-thunk';
+import axios from 'axios';
+import contants from '../Redux/constants';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+const middleware = [thunk];
+const mockStore = configureStore(middleware);
+
+Enzyme.configure({
+	adapter: new Adapter(),
+});
+jest.mock('axios');
+
+let store, wrapper;
+const successState = {
+	Verify: {
+		pending: false,
+		verifyResult: {
+			msg: 'verified',
+			status: 200,
+		},
+		error: null,
+	},
+};
+const initialState = {
+	Verify: {
```

Comment:
hi @lemoissonM great job but let us try to be specific in our tests and categorize `action tests`, `reducers tests` and `components tests` and we create a folder for those like redux within our test folder